vision/module_in_frame/ModuleInFrame.py

```diff
@@ -0,0 +1,68 @@
+"""
+This file contains the ModuleInFrame function to detect if the module is in an image
+"""
+
+import cv2
+import numpy as np
+
+def ModuleInFrame(imgPath):
+    """
+    Determines if the Module is in frame
+
+    Parameters
+    ----------
+    imgPath, is a string that is the path of the image to be passed into the function
+
+    Returns
+    -------
+    isInFrame, a bool, which is true if the module is in the frame and false if not in the frame
+    """
+
+    # Ignore numpy warnings
+    np.seterr(all="ignore")
+
+    # Get target image
+    img = cv2.imread(imgPath)
+
+    # Remove depth channel
+    img.reshape((-1, 3))
```

Comment:
This will string out and jumble up the image instead of removing the depth. -- also this is running the function but not overriding the data in img, which is why the code still works.

```
img = img[:, :, :3]  # This forces the image to 3 channels.
```